db/schema.rb

```diff
@@ -0,0 +1,29 @@
+# encoding: UTF-8
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# Note that this schema.rb definition is the authoritative source for your
+# database schema. If you need to create the application database on another
+# system, you should be using db:schema:load, not running all the migrations
+# from scratch. The latter is a flawed and unsustainable approach (the more migrations
+# you'll amass, the slower it'll run and the greater likelihood for issues).
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema.define(version: 20160824165957) do
+
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
+  create_table "education_benefits_claims", force: :cascade do |t|
+    t.datetime "submitted_at"
+    t.datetime "processed_at"
+    t.json     "form",         null: false
```

Comment:
What level of PII is in this form? Parts of it may need to be encrypted. We'll eventually have a form service that handles stuff like encryption and field commonality - which this may be the start of - just want to make sure we're not storing SSN, bank account #s in plain text.